### DISCUSSION TITLE
Mengubah untitled Rscript Baru menjadi global.R

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#574] [Q&A] [answered]
Halo, saya mau tanya Bagaimana mengubah untitled Rscript baru menjadi global.R

**Comment by idoali** [comment, marked as answer]
Selamat pagi pak Moses, terima kasih atas pertanyaannya

Bapak bisa melakukan **Save As** kemudian file diberi nama `global.R`. Atau bisa juga di rename melalui finder seperti file biasa.
